### DISCUSSION TITLE
cmake: command not found

### DISCUSSION THREAD

**Discussion by Quincy-QC** [Tencent/libpag#2333] [Q&A] [answered]
【版本信息】

Apple M1 Pro
MacOS 14.4.1 (23E224)
Xcode Version 15.2 (15C500b)

【平台信息】

iOS 原生

【预期的表现】

编译通过

【实际的情况】

编译报错
![lNEGYzIOif](https://github.com/Tencent/libpag/assets/18659596/acef272d-d360-4f54-8536-7ebe4b86155b)

本地cmake版本：3.29.4

**Comment by kevingpqi123** [comment, marked as answer]
基础问题看下根目录下的 README 和 ios 目录下的 README，请严格按照上面的要求执行，我们的每一次提交都会基于 github action 的自动化测试
